spk/postgresql/src/installer.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/sh
+
+# Package
+PACKAGE="postgresql"
+DNAME="PostegreSQL"
```

Comment:
These variables are never used (replaced by SYNOPKG_PKGNAME), correct? Is so please remove these lines.